lib/miq_automation_engine/service_models/mixins/miq_ae_service_miq_request_mixin.rb

```diff
@@ -4,10 +4,12 @@ def options
   end
 
   def get_option(key)
+    reload
```

Comment:
I am against adding the `reload` to the `get_*` methods, this could introduce a large performance overhead for minimal gain.  Users still have the option to call `reload` directly on the object if needed.

I would suggest holding off on this one as it is not needed to resolve this issue.